### PR TITLE
Make AI switch weapons at random

### PR DESCRIPTION
**Pull request by soryy708**
:cl: Make AI switch weapons at random
add: Added ability for AI ships to switch weapons
/:cl:

### CODE REVIEW

File: DS13/code/modules/overmap/ai.dm [hunk 1]
```diff
@@ -145,7 +145,17 @@
 		nav_target = target
 		if(cloaked) //No firing while cloaked, please!
 			return
+		
+		var/shouldSwitchWeapons = rand(0, 100)
+		if(shouldSwitchWeapons > 66) // 1 in 3 chance of switching weapons every 2 seconds
```
Comment (Kmc2000) [inline DS13/code/modules/overmap/ai.dm]: if(prob(33))

File: DS13/code/modules/overmap/ai.dm [hunk 2]
```diff
@@ -145,7 +145,17 @@
...
+			if(fire_mode == "phaser")
+				if(photons > 0)
+					fire_mode = "torpedo"
```
Comment (Kmc2000) [inline DS13/code/modules/overmap/ai.dm:154]: Is this the right name? can't remember if it's that or photon

**Review by Kmc2000 — CHANGES_REQUESTED** [review]

**Comment by Kmc2000** [comment]
You also need to change their photons var so they start with some.

File: DS13/code/modules/overmap/ai.dm (hunk 2 above)
Comment (soryy708) [reply DS13/code/modules/overmap/ai.dm:154]: The attack code checks if it's a phaser, else it assumes torpedo.

**Comment by Kmc2000** [comment]
Lemme know when this is all tested and ready

**Comment by soryy708** [comment]
@Kmc2000 AI ships seem to never target me now. Any idea why this could happen?

**Comment by Kmc2000** [comment]
I've integrated this into my own AI improvements

**Pull request closed without merging**